src/components/About/About.jsx

```diff
@@ -0,0 +1,7 @@
+export default function About() {
+  return (
+    <div>
+      <p>About</p>
+    </div>
```

Comment:
This file has a `<div>` around the content but it's just a `<>` in Home - this is probably fine but might be worth making it consistent